src/views/Header.js

```diff
@@ -80,6 +81,17 @@ class Header extends React.PureComponent<void, HeaderProps, HeaderState> {
     ).options.headerTruncatedBackTitle;
   }
 
+  _getBackButtonTitleStyle(scene: NavigationScene): ?Style {
+    const lastScene = this._getLastScene(scene);
+    const { headerBackTitleStyle } = this.props.getScreenDetails(scene).options;
+    if (!lastScene) {
+      return headerBackTitleStyle;
+    }
+    const { headerTitleStyle } = this.props.getScreenDetails(lastScene).options;
+
+    return [headerTitleStyle, headerBackTitleStyle];
```

Comment:
Also font family probably makes sense, but there are so many other styles you don't wanna inherit. I'm interested in what do other iOS apps do